.github/workflows/launcher_go.yml

```diff
@@ -2,6 +2,8 @@ name: launcher_go
 
 on:
   push:
+    tags:
+      - "v*.*.*"
```

Comment:
I don't think we should be using tags here to push out a release. This repository contains more than just `launcher` so a git tag would make little sense for the `discourse_docker` related stuff. Are we able to use some sort of version file here so that a new release is created when the version in the file is bumped?

Reply:
sure let me update to watch for that 👍

Reply:
So, short answer no it's not going to work exactly as you requested. But I've updated it so we push assets to a single release.

Github releases require a git tag, and if we don't want to provide one, we need to create a dummy tag for the release to pin to, and create a single `launcher2` release. We can then overwrite assets within that release, and that single release becomes our go-to 'latest' release for the assets.